src/renderer/state/test/mocks.ts

```diff
@@ -460,8 +496,38 @@ export const mockWorkingUploadJob: UploadJob = {
 };
 
 export const mockWaitingUploadJob: UploadJob = {
-  ...mockWorkingUploadJob,
+  created: new Date(),
+  currentStage: "Waiting",
+  jobId: "2222222222",
+  jobName: "mockWaitingUploadJob",
```

Comment:
I'm having trouble finding what previous commit this mock code (lines 499 to 530) came from. The [original PR](https://github.com/aics-int/aics-file-upload-app/pull/34/files#diff-f028817c73f3e18d69f2fe44e8c75c213f6ab26dd9f8e587f5c12528b63ac0f6) just added the `mockSuccessfulUploadJobWithUnexposedAnnotation` mock and left this one untouched as follows:
```javascript
export const mockWaitingUploadJob: JSSJob = {
  ...mockWorkingUploadJob,
  currentStage: UploadStage.WAITING_FOR_CLIENT_COPY,
  status: JSSJobStatus.WAITING,
};
```
(Note though that `main-FSS2` doesn't have the `currentStage` line.)

Reply:
That would track - I believe this was the one test I had to edit manually to get it to play nice with a typing change.